content/docs/command-reference/repro.md

```diff
@@ -13,7 +13,7 @@ usage: dvc repro [-h] [-q | -v] [-f] [-s] [-c <path>] [-m] [--dry] [-i]
                  [--no-commit] [--downstream] [targets [targets ...]]
 
 positional arguments:
-  targets        Stage or .dvc file to reproduce. 'Dvcfile' by default.
+  targets        Stage to reproduce.
```

Comment:
"or .dvc file" should stay.

Reply:
But now the pipeline information is being kept in `dvc.yaml` file and I have seen that no `.dvc` file is being generated when you execute `dvc run` but a new entry is added to `dvc.yaml` file. So how the target would be a .dvc file? 

Rather we can say like -
`Stage to reproduce or one of the entry under dvc.yaml file`.

Reply:
Nope. That suggestion is redundant. Again, the original text is correct: "Stage or .dvc file to reproduce" but without period `.` BTW.

Reply:
>  but without period `.`

You mean it should be -  `Stage or DVC file to reproduce` ?